userguide/virtualmachines.rst

```diff
@@ -156,83 +156,69 @@ a Virtual Machine (VM) type are described in
    | 4        | Adapter Type       | drop-down menu | :guilabel:`Intel e82545 (e1000)` emulates the same Intel Ethernet card. This                  |
    |          |                    |                | provides compatibility with most operating systems. :guilabel:`VirtIO` provides               |
    |          |                    |                | better performance when the operating system installed in the VM supports VirtIO              |
    |          |                    |                | paravirtualized network drivers.                                                              |
-   |          |                    |                |                                                                                               |
    +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
    | 4        | MAC Address        | string         | Enter the desired MAC address to override the auto-generated                                  |
    |          |                    |                | randomized MAC address.                                                                       |
-   |          |                    |                |                                                                                               |
    +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
    | 4        | Attach NIC         | drop-down menu | Select the physical interface to associate with the VM.                                       |
-   |          |                    |                |                                                                                               |
-   |          |                    |                |                                                                                               |
    +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
-   | 5        | Optional: Choose   | browse button  | Click |ui-browse| to select an installer ISO or image file on the %brand% system.             |
-   |          | installation media |                |                                                                                               |
-   |          | image              |                |                                                                                               |
+   | 5        | Optional: Choose   | browse button  | Click |ui-browse| to select an installer ISO or image file on the %brand%                     |
+   |          | installation media |                | system. After the media has been installed, delete the media device. Go to                    |
```

Comment:
I don't think including this in the table here is the way to do it. Instead, add it in the text at the end of installing the VM.